client/post-editor/editor-more-options/copy-post.jsx

```diff
@@ -58,54 +56,63 @@ class EditorMoreOptionsCopyPost extends Component {
 	}
 
 	goToNewDraft = () => {
-		if ( '' !== this.state.selectedPostId ) {
-			page.redirect( `/post/${ this.props.siteSlug }?copy=${ this.state.selectedPostId }` );
+		const { siteSlug, type } = this.props;
+		const { selectedPostId } = this.state;
+		if ( '' !== selectedPostId ) {
+			page.redirect( `/${ type }/${ siteSlug }?copy=${ selectedPostId }` );
 			this.closeDialog();
 		}
 	}
 
 	render() {
-		const { translate, siteId } = this.props;
+		const { siteId, translate, type } = this.props;
+		const { selectedPostId, showDialog } = this.state;
 		const buttons = [ {
 			action: 'cancel',
 			label: translate( 'Cancel' ),
 		}, {
 			action: 'copy',
 			label: translate( 'Overwrite' ),
 			isPrimary: true,
-			disabled: ! this.state.selectedPostId,
+			disabled: ! selectedPostId,
 			onClick: this.goToNewDraft,
 		} ];
 		return (
 			<AccordionSection className="editor-more-options__copy-post">
 				<EditorDrawerLabel
-					labelText={ translate( 'Copy Post' ) }
-					helpText={ translate( "Pick a post and we'll copy the title, content, tags and categories. " ) }
+					labelText={ this.isPost() ? translate( 'Copy Post' ) : translate( 'Copy Page' ) }
+					helpText={ this.isPost()
+						? translate( "Pick a post and we'll copy the title, content, tags and categories." )
+						: translate( "Pick a page and we'll copy the title, content, tags and categories." )
```

Comment:
@copons I think this needs to be tweaked since IIRC you can't add tags and categories to a Page.

`Pick a page and we'll copy the title and content.`

or maybe:

`Pick a page and we'll copy the title and content to a new one.`?